### ISSUE TITLE
Comment booking

### ISSUE THREAD

**Issue by Agetorp** [davidjosefson/mah-book-room-api#7]
In Kronox you have the possibility to add a comment to your booking. I miss that in this system.

**Comment by davidjosefson** [comment]
True that!
Row 155 in routes/bookings.js:

``` javascript
var url = 'https://schema.mah.se/ajax/ajax_resursbokning.jsp?op=boka&datum=' + date + '&id=' + room + '&typ=RESURSER_LOKALER&intervall=' + time + '&moment= &flik=FLIK-0017';
```

The property `moment` is what needs to be changed dynamically to what ever comment the user wants.

**Comment by davidjosefson** [comment]
Fixed in release v0.3.0.

To include a comment to a booking, add the `comment` attribute in POST to /bookings, like this:

``` json
{
    "room": "nia0402",
    "time": "17152000",
    "date": "2015-12-25",
    "comment": "This is an optional comment!"
}
```
